app/code/Magento/Catalog/Block/Adminhtml/Category/Tab/Product.php

```diff
@@ -159,6 +165,30 @@ protected function _prepareColumns()
         );
         $this->addColumn('name', ['header' => __('Name'), 'index' => 'name']);
         $this->addColumn('sku', ['header' => __('SKU'), 'index' => 'sku']);
+
```

Comment:
Static tests are failing due to this: Code must not contain multiple empty lines in a row; found 2 empty lines